features/speech-recognition-grammar.yml

```diff
@@ -0,0 +1,23 @@
+name: Speech recognition grammar
+description: "The `SpeechRecognition.grammars` property of the Web Speech API represents a set of words or patterns of words that the recognition service should recognize, used for when the recognition service's accuracy is low. This feature has been removed due to advancements in speech recognition."
+spec: https://webaudio.github.io/web-speech-api/#speechreco-section
+discouraged:
+  # SpeechGrammar and SpeechGrammarList have been removed from the Web Speech API spec.
+  # There is no alternative API to point to because speech recognition has improved
+  # significantly, removing the need for grammar-based recognition.
+  according_to:
+    - https://github.com/WebAudio/web-speech-api/pull/117
```

Comment:
It's interesting that they simply removed this from the spec, as opposed to marking as discouraged.

Reply:
I thought too. I guess the thing never quite picked up interest and they felt it safe to just remove ...